src/helpers/getCommentsById.js

```diff
@@ -0,0 +1,3 @@
+export const getCommentsById = (commentsFromServer, postId) => (
```

Comment:
The function name 'getCommentsById' does not accurately represent its functionality. It filters comments by post ID, not comment ID. A more accurate name might be 'getCommentsByPostId'